docs/tutorial/quick-start.md

```diff
@@ -0,0 +1,84 @@
+# Quick start
+
+## What you'll do
+
+- Setup a GitHub repository
+- Activate the GitHub APIs related to self-hosted runner
+- Deploy the [GitHub runner charm](https://charmhub.io/github-runner)
+- Ensure GitHub repository setting are secure
+- Run a simple workflow on the self-hosted runner
+
+## Requirements
+
+- GitHub Account.
+- Juju 3 installed.
+- Juju controller on OpenStack, and a juju model.
```

Comment:
Wouldn't it make more sense to use LXD for this tutorial (and remove the LXD howto)? According to the [Diátaxis](https://diataxis.fr/) framework, tutorials are intended for learning experiences (not real-world deployments), and I think more users can quickly spawn a local lxd than having a juju controller in an OpenStack environment.